HelpMyStreetFE/HelpMyStreetFE/Services/Requests/FilterService.cs

```diff
@@ -201,6 +283,50 @@ private SortAndFilterSet GetCompletedRequestsDefaultSortAndFilterSet()
             return filterSet;
         }
 
+        public IEnumerable<ShiftJob> SortAndFilterJobs(IEnumerable<ShiftJob> jobs, JobFilterRequest jfr)
+        {
+
+
+            var jobsToDisplay = jobs.Where(
+                j => (jfr.JobStatuses == null || jfr.JobStatuses.Contains(j.JobStatus))
+                    && (jfr.SupportActivities == null || jfr.SupportActivities.Contains(j.SupportActivity))
+                    && (jfr.Locations == null || jfr.Locations.Contains(j.Location))
+                    && (jfr.DueInNextXDays == null || j.StartDate <= DateTime.Now.Date.AddDays(jfr.DueInNextXDays.Value))
```

Comment:
I don't think we're populating Due Date with anything useful for shifts.

Reply:
I'm not... using that... (just the DueInNextXDays Enum)... I think!?

Reply:
Right you are.